### PR TITLE
Elvis/unidax exchange intergration

### PR DESCRIPTION
**Pull request by ElvisLYC**
- What is the purpose of this Pull Request?
- What is the related issue for this Pull Request (if this PR fixes issue, prepend with "Fixes" or "Closes")?
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly
- [x] I have implemented the `trade_page_url` method that links to the exchange page with the `base` and `target` passed in. If not available, enter the root domain of the exchange website.
- [x] I have verified at least **ONE** ticker volume matches volume shown on the trading page (use script below)

```
client = Cryptoexchange::Client.new
pairs = client.pairs 'exchange_name'
tickers = pairs.map do |p| client.ticker p end
sorted_tickers = tickers.sort_by do |t| t.volume end.reverse
```

### CODE REVIEW

File: lib/cryptoexchange/exchanges/unidax/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Unidax::Market::NAME
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.high      = NumericHelper.to_d(output['high'])
+          ticker.low       = NumericHelper.to_d(output['low'])
+          ticker.bid       = NumericHelper.to_d(output['buy'])
+          ticker.ask       = NumericHelper.to_d(output['sell'])
+          ticker.volume    = NumericHelper.to_d(output['vol'])
+          ticker.timestamp = NumericHelper.to_d(output['time']) / 1000
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/unidax/services/market.rb]: lets set this to `nil`, as we faced tonnes of issue relying on the returned timestamp

File: README.md [hunk 2]
```diff
@@ -360,6 +360,7 @@ Or install it yourself as:
 | TRXMarket         | Y       |            |         |         | Y           |          | trx_market        |       |
 | Txbit             | Y       |            |         |         | Y           | Y        | txbit             |       |
 | UEX               | Y       |            |         |         | Y           |          | uex               |       |
+| Unidax            | Y       | N          | N       |         | Y           | N        | unidax           |       |
```
Comment (tmlee) [inline README.md]: align the last column

Comment (ElvisLYC) [reply README.md]: done

File: lib/cryptoexchange/exchanges/unidax/services/market.rb (hunk 1 above)
Comment (ElvisLYC) [reply lib/cryptoexchange/exchanges/unidax/services/market.rb]: done

**Pull request merged**